tox.ini

```diff
@@ -40,4 +40,5 @@ deps =
     mypy
     types-click
     types-requests
+    types-jsonschema
```

Comment:
FYI you can run mypy --install-types to sanity check nothing else is missing. Everything else I'll have to get to tomorrow - sorry I missed this earlier.

Reply:
Yep, mypy is kind enough to inform you of that any time you run it :slightly_smiling_face: 

```
cvelib/cve_api.py:9: error: Library stubs not installed for "jsonschema"  [import]
cvelib/cve_api.py:9: note: Hint: "python3 -m pip install types-jsonschema"
cvelib/cve_api.py:9: note: (or run "mypy --install-types" to install all missing stub packages)
cvelib/cve_api.py:9: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
```